lib/tasks/reviews.rake

```diff
@@ -0,0 +1,80 @@
+require 'json'
+require 'net/http'
+require 'optparse'
+require 'uri'
+require 'reverse_markdown'
+namespace :reviews do
+  desc 'Imports review from NRDBc, currently storing usernames as strings instead of references'
+
+  def retrieve_reviews
+    url = URI('https://netrunnerdb.com/api/2.0/public/reviews')
+
+    http = Net::HTTP.new(url.host, url.port)
+    http.use_ssl = true
+    request = Net::HTTP::Get.new(url)
+
+    response = http.request(request)
+
+    return JSON.parse(response.body) if response.is_a?(Net::HTTPSuccess)
+
+    raise "Failed to retrieve decklists! Status code: #{response.code}"
+  end
+
+  def purge_tables
+    # Only do this in a transaction
+    puts 'Purging Review Tables'
+    ReviewVote.delete_all
+    ReviewComment.delete_all
+    Review.delete_all
+  end
+
+  task import: :environment do
+    puts 'Importing Reviews from NetrunnerDB Classic'
+    reviews_body = retrieve_reviews
+    Review.transaction do
+      purge_tables
+      puts 'Starting import'
+      reviews_body['data'].each do |review|
+        card_name = review['title']
+        rev_body = ReverseMarkdown.convert review['ruling']
+        username = review['user']
+        comments = review['comments']
+        card = Card.find_by(title: card_name)
```

Comment:
i would actually re-use the text_to_id function from the card import and do a straight find instead.  You could make a quick map with `Card.all.pluck(:id).to_set` and check against that instead of doing a full card load each time.